elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -98,30 +104,30 @@ $pfe-modal--breakpoint--medium: 640px;
 	}
 
 	&__close {
-		@include pfe-reset-button;
+		@extend %reset-button;
 		position: absolute;
-		top: calc(#{pfe-var(container-padding)} * .25);
-		right: calc(#{pfe-var(container-padding)} * .25);
+		top: 	calc(#{pfe-var(container-padding)} * .25);
+		right: 	calc(#{pfe-var(container-padding)} * .25);
 		cursor: pointer;
 		line-height: .5;
-		padding: pfe-var(container-padding);
+		padding: #{pfe-var(container-padding)};
 
 		@media screen and (min-width: $pfe-modal--breakpoint--medium) {
-			top: #{pfe-var(container-padding)};
-			right: #{pfe-var(container-padding)};
+			top: 	#{pfe-var(container-padding)};
+			right: 	#{pfe-var(container-padding)};
 		}
 
 		@media screen and (max-height: $pfe-modal--breakpoint--medium) and (min-width: $pfe-modal--breakpoint--medium) {
-			top: calc(#{pfe-var(container-padding)} / 2);
-			right: calc(#{pfe-var(container-padding)} / 2);
+			top: 	calc(#{pfe-var(container-padding)} / 2);
+			right: 	calc(#{pfe-var(container-padding)} / 2);
 		}
 
 		> svg {
-			fill: pfe-radio(text);
-			height: pfe-var(ui--element--size);
-			width: pfe-var(ui--element--size);
-			height: pfe-local($cssvar: size, $region: close);
-			width: pfe-local($cssvar: size, $region: close);
+			fill: 	#{pfe-apply-broadcast(text)};
+			height: #{pfe-var(ui--element--size)};
+			width: 	#{pfe-var(ui--element--size)};
+			height: #{pfe-local($cssvar: size, $region: close)};
+			width: 	#{pfe-local($cssvar: size, $region: close)};
```

Comment:
Do you know why the double declarations with different values, is this for IE or Edge? A comment would be good here.  cc @chrisdo1

Reply:
I didn't change anything - just added interpolation so that seems out of scope.  Not related to updates on this branch.